modules/graphql/server-ts/types.ts

```diff
@@ -0,0 +1,71 @@
+import { GraphQLSchema } from 'graphql';
+import { Server } from 'http';
+import { SubscriptionServer } from 'subscriptions-transport-ws';
+import ServerModule from '@gqlapp/module-server-ts';
+
+/**
+ * A function to make a GraphQL context creator
+ *
+ * @param modules App modules
+ *
+ * @returns function that creates a GraphQL context
+ */
+export type MakeGQLContextCreator = (modules: ServerModule) => (req: Request, res: Response) => any;
```

Comment:
Types must be declared closer to the locations where they are used. It is a very bad practice to use `MakeGQLContextCreator` in only a single place and move declaration of the type to another place.

Reply:
Fixed.

Reply:
@basilalex What about all the other types?